### PR TITLE
Make `UISlider` great again

### PR DESCRIPTION
**Pull request by eimantas**
- Make `value` reactive;
- Make `minimumValue` reactive;
- Make `maximumValue` reactive;

### CODE REVIEW

**Comment by andersio** [comment]
What about the signal?

**Comment by eimantas** [comment]
Should the signal be called `values` or `continuousValues`?

**Comment by andersio** [comment]
I'd name it `values`, which fits well in both cases, if we aren't interested in providing both variants.

File: ReactiveCocoaTests/UIKit/UISliderSpec.swift [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+		it("should emit user's changes for its value") {
+			slider.value = 0.25
+
+			var updatedValue: Float?
+			slider.reactive.values.observeValues { value in
+				updatedValue = value
+			}
+
+			slider.sendActions(for: .valueChanged)
```
Comment (andersio) [inline ReactiveCocoaTests/UIKit/UISliderSpec.swift]: May you assert `updatedValue` to be `nil` before triggering the action?

**Comment by altyus** [comment]
😢  but only because of the PR name

**Comment by andersio** [comment]
🙁

**Pull request merged**